<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>700</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>700</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Cam Shaft</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QRadioButton" name="MU">
        <property name="text">
         <string>Metric Units </string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="IU">
        <property name="text">
         <string>Imperial Units</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Harmonic Motion</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <property name="leftMargin">
           <number>3</number>
          </property>
          <property name="topMargin">
           <number>3</number>
          </property>
          <property name="rightMargin">
           <number>3</number>
          </property>
          <property name="bottomMargin">
           <number>3</number>
          </property>
          <item>
           <widget class="QGroupBox" name="groupBox_9">
            <property name="title">
             <string>ΔR</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_17">
             <item>
              <layout class="QVBoxLayout" name="Har_R"/>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="groupBox_10">
            <property name="title">
             <string>velocity</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_15">
             <item>
              <layout class="QVBoxLayout" name="Har_velocity"/>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="groupBox_11">
            <property name="title">
             <string>acceleration</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_19">
             <item>
              <layout class="QVBoxLayout" name="Har_acceleration"/>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="groupBox_12">
            <property name="title">
             <string>jerk</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_18">
             <item>
              <layout class="QVBoxLayout" name="Har_jump"/>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QSlider" name="HarThita">
            <property name="maximum">
             <number>360</number>
            </property>
            <property name="value">
             <number>360</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>Cycloidal motion</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <property name="leftMargin">
           <number>3</number>
          </property>
          <property name="topMargin">
           <number>3</number>
          </property>
          <property name="rightMargin">
           <number>3</number>
          </property>
          <property name="bottomMargin">
           <number>3</number>
          </property>
          <item>
           <widget class="QGroupBox" name="groupBox_17">
            <property name="title">
             <string>ΔR</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_24">
             <item>
              <layout class="QVBoxLayout" name="Cy_R"/>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="groupBox_18">
            <property name="title">
             <string>velocity</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_25">
             <item>
              <layout class="QVBoxLayout" name="Cy_velocity"/>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="groupBox_19">
            <property name="title">
             <string>acceleration</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_26">
             <item>
              <layout class="QVBoxLayout" name="Cy_acceleration"/>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="groupBox_20">
            <property name="title">
             <string>jerk</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_27">
             <item>
              <layout class="QVBoxLayout" name="Cy_jump"/>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QSlider" name="CyThita">
            <property name="maximum">
             <number>360</number>
            </property>
            <property name="value">
             <number>360</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
